doc/en/components/layouts/tile-manager.md

```diff
@@ -0,0 +1,320 @@
+| `title` | Content for the tile header. |
+| `fullscreen-action` | Overwrite the default fullscreen action content. |
+| `maximize-action` | Overwrite the default maximize action content. |
+| `actions` | 	Custom content rendered after the default actions. |
+| `side-adorner` | Overwrite the default horizontal resize adorner. |
+| `corner-adorner` | 	Overwrite the default diagonal resize adorner. |
+| `bottom-adorner` | Overwrite the default vertical resize adorner. |
+
+
+### Header section actions
+
+By default, the header section includes two action buttons:
+
+- The `maximize` button enlarges the tile's content to fill the entire width of the Tile Manager, offering a wider view of the content.
+- The `fullscreen` button enables the tile to open in fullscreen mode in the user's browser.
+
+<img src="../../images/tile-manager-actions.png" />
+
+If you want to display just one of the two buttons, you can set it as a slot attribute within the Tile component. Use the `maximize-action` value to show only the maximize button, or the `fullscreen-action` value to show only the fullscreen button.
+
+```html
+<igc-tile-manager>
+  <igc-tile>
+    <div slot="maximize-action"></div>
+    <p>Content for Tile 1</p>
+  </igc-tile>
+</igc-tile-manager>
+```
+
+You also have the option to disable both action buttons and create custom ones according to your preferences.
+
+`sample="/layouts/tile-manager/actions", height="700", alt="{Platform} Tile Manager Actions Example"`
+
+In this example, we created custom action buttons using the Ignite UI Icon Button component.
+
+## Resizing
+
+Resizing in the Tile Manager is a functionality that allows tiles to be resized using three different resize adorners.
+
+- **Side Adorner**: Adjusts the width by modifying the column span.
+- **Bottom Adorner**: Adjusts the height by modifying the row span.
+- **Corner Adorner**: Adjusts both width and height simultaneously.
+
+To ensure smooth resizing, a ghost element is used instead of directly modifying the tile’s dimensions. This element appears on top of the original tile, displaying its current dimensions when resizing begins, and it updates in real time as the user drags any of the resize handles.
+
+> [!Note] If the ghost element exceeds the available grid space, it will automatically adjust to the largest possible span within the grid's limits.
+
+The Tile Manager automatically rearranges itself when a tile changes size, ensuring that there is minimal empty space. That's why expanding a tile may push adjacent tiles into new positions, while shrinking creates gaps that other tiles may fill dynamically. This ensures that the Tile Manager stays as compact as possible, without any overlapping tiles, and that all movements remain within the defined grid structure.
+
+We can use the `resize-mode` property to control how resizing is applied in the Tile Manager. It can be set to either `hover` or `always`, which determines when the resize adorners are visible.
```

Comment:
```suggestion
We can use the `resize-mode` property to control how resizing is applied in the Tile Manager. It can be set to `none`, `hover` or `always`, which determines when the resize adorners are visible. Default value is `none` and the tile could not be resized.
```